src/scripts/main.js

```diff
@@ -1,3 +1,68 @@
 'use strict';
 
-// write code here
+const field = document.querySelector('.field');
+const table = field.firstElementChild;
+const rows = table.rows;
+const cells = rows[0].cells;
+const removeRow = document.querySelector('.remove-row');
+const appendRow = document.querySelector('.append-row');
+const removeColumn = document.querySelector('.remove-column');
+const appendColumn = document.querySelector('.append-column');
```

Comment:
Відступи не будуть зайвими
```suggestion
const field = document.querySelector('.field');
const table = field.firstElementChild;

const rows = table.rows;
const cells = rows[0].cells;

const removeRow = document.querySelector('.remove-row');
const appendRow = document.querySelector('.append-row');
const removeColumn = document.querySelector('.remove-column');
const appendColumn = document.querySelector('.append-column');
```